test/end-to-end-tests/src/@types/global.d.ts

```diff
@@ -15,16 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-module.exports = class LogBuffer {
-    constructor(page, eventName, eventMapper, reduceAsync=false, initialValue = "") {
-        this.buffer = initialValue;
-        page.on(eventName, (arg) => {
-            const result = eventMapper(arg);
-            if (reduceAsync) {
-                result.then((r) => this.buffer += r);
-            } else {
-                this.buffer += result;
-            }
-        });
+declare global {
```

Comment:
Would appreciate some help with this as it doesn't quite work - references to `window.mxPerformanceMonitor` still trigger type errors, which I'm working around by calling it `window['mxPerformanceMonitor']`

Reply:
I'm a bit surprised to see e2e-test specific declarations tbh. In any case, the types will need to be imported from somewhere central (or each test if not central) - I'd recommend importing the real types from the react-sdk src instead.

If importing the real ones causes issues (like import loops), then comments on the `any` usage would be preferred.